File: Queue/QueueJava/Node.java
package Queue.QueueJava;

public class Node {
    private char dato;
    private Node next;
    
    public Node(char dato) {
        this.dato = dato;
    }
    
    public char getDato() {
        return this.dato;
    }
    
    public Node getNode() {
        return next;
    }
    
    public void setNode(Node next) {
        this.next = next;
    }
}

File: Queue/QueuePython/Node.py

class Node:

    __dato = None
    __node = None
    
    def __init__(self, dato):
        self.__dato = dato
    
    def setDato(self, dato):
        self.__dato = dato
    
    def getDato(self):
        return self.__dato
    
    def setNode(self, node):
        self.__node = node

    def getNode(self):
        return self.__node